src/importexport/imagesexport/internal/svgwriter.cpp

```diff
@@ -99,7 +99,8 @@ mu::Ret SvgWriter::write(INotationPtr notation, QIODevice& destinationDevice, co
 
     mu::engraving::MScore::pixelRatio = mu::engraving::DPI / printer.logicalDpiX();
 
-    if (!options[OptionKey::TRANSPARENT_BACKGROUND].toBool()) {
+    const bool TRANSPARENT_BACKGROUND = Val(configuration()->exportSvgWithTransparentBackground()).toBool();
+    if (!TRANSPARENT_BACKGROUND) {
```

Comment:
Since `configuration()->exportSvgWithTransparentBackground()` is already just a bool, there is no need to convert back-and-forth to `Val`. So you can do this:
```suggestion
    if (!configuration()->exportSvgWithTransparentBackground()) {
```
(same applies in PngWriter)

Reply:
> Since `configuration()->exportSvgWithTransparentBackground()` is already just a bool, there is no need to convert back-and-forth to `Val`. So you can do this:
> 
> (same applies in PngWriter)

You're right, I'll edit that. What's the preferred workflow for that, amending the original commit or adding a new one to the branch/pull request? More used to gerrit :)

Reply:
Either way works, but I'd rather amend the original comit